src/assets/styles/theme.scss

```diff
@@ -5,33 +5,33 @@
  */
 @import '~normalize.css/normalize';
 
-/* Utils */
-@import './tools/mixins';
+// Settings
+@import './variables';
 
-/* Settings */
-@import './settings/variables';
+// Tools
+@import './tools/mixins';
 
-/* Global */
-@import './global/slate-reset';
-@import './global/helper-classes';
-@import './global/layout';
-@import './global/icons';
-@import './global/typography';
-@import './global/rte';
-@import './global/responsive-tables';
-@import './global/forms';
+// Core
```

Comment:
What structure of the comment do we have to follow? `//`, `/* */` or `/* ================ Comment ================ */`?

Reply:
Think we can go simple and add `//` for every comment... personally, I'm not a fan of the large `/* === ... === */` formatted comments.

Reply:
Sounds good 👍